docs/operator_guides/0_running_an_operator.md

```diff
@@ -70,15 +73,22 @@ To build the operator binary, run:
 make build_operator
 ```
 
-To update the operator, run:
+### Upgrading the Operator
+
+If you want to upgrade the operator, you need checkout the latest version of the repository and rebuild the operator.
 
 ```bash
-git pull
+git fetch
+git checkout v0.2.1 # or <latest_version>
 make build_operator
```

Comment:
what do you think about using something like what appears here
https://stackoverflow.com/questions/17414104/git-checkout-latest-tag
and make it just one command?

Reply:
I tihnk you have to make a fetch anyways